### PR TITLE
feature-32: Change default settings for background color, invoice attachments and name scheme

### PR DESCRIPTION
**Pull request by lcduong**
> Fixes [#32](https://github.com/fossasia/eventyay-tickets/issues/32)
> 
> #### Short description of what this resolves:
> Change default settings for background color, invoice attachments and name scheme
> 
> #### Changes proposed in this pull request:
> * Change default settings for background color, invoice attachments and name scheme
> 
> #### Checklist
> * [x]  I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
> * [x]  My branch is up-to-date with the Upstream `development` branch.
> * [ ]  The acceptance, integration, unit tests and linter pass locally with my changes
> * [ ]  I have added tests that prove my fix is effective or that my feature works
> * [ ]  I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by lcduong** [comment]
Hi @mariobehling @norbusan , changes for default background color is done, please help me to review and merge it.

image

image

File: src/pretix/static/pretixpresale/scss/main.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/* before variables.scss because it only affects presale, not control */
```
Comment (mariobehling) [inline src/pretix/static/pretixpresale/scss/main.scss]: /* before variables.scss as it affects presale only, not control */

File: src/pretix/base/models/event.py [hunk 2]
```diff
@@ -453,6 +453,8 @@ def set_defaults(self):
         self.settings.ticketoutput_pdf__enabled = True
         self.settings.ticketoutput_passbook__enabled = True
         self.settings.event_list_type = 'calendar'
+        self.settings.invoice_email_attachment = True
+        self.settings.name_scheme = 'given_family'
```
Comment (mariobehling) [inline src/pretix/base/models/event.py:457]: I think link 457 should be added in a follow up PR.

**Review by mariobehling — CHANGES_REQUESTED** [review]

File: src/pretix/static/pretixpresale/scss/main.scss (hunk 1 above)
Comment (lcduong) [reply src/pretix/static/pretixpresale/scss/main.scss]: Hi @mariobehling, text is updated.

File: src/pretix/base/models/event.py (hunk 2 above)
Comment (lcduong) [reply src/pretix/base/models/event.py:457]: Hi @mariobehling , did not get your point, name_schema 'given_family' is default for new event, what need to be change?

**Review by mariobehling — APPROVED** [review]

**Pull request merged**